src/test/java/core/basesyntax/service/OutputDataServiceImplTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+import org.junit.Test;
+
+public class OutputDataServiceImplTest {
+    @Test
+    public void toStringConverter_convertCorrectData_ok() {
+        OutputDataService outputDataService = new OutputDataServiceImpl();
```

Comment:
Make tested service class-level field - you use it in all methods.

**Same for other test classes.**